servo/types.py

```diff
@@ -371,6 +371,14 @@ class Unit(str, enum.Enum):
     """Application throughput in terms of requests processed per minute.
     """
 
+    REQUESTS_PER_MINUTE_WF = "requests/m"
+    """Application throughput in terms of requests processed per minute in Wavefront format.
+    """
+
+    ERRORS_PER_MINUTE_WF = "errors/m"
+    """Application error rate in terms of errors per minute in Wavefront format.
+    """
+
```

Comment:
So I am confused how to implement this change. Might need a sample code hint! Units can't be subclassed as an Enum right?

Reply:
There are also some more possible types we need to take in from wavefront once I understand the ideal way to do so

Reply:
Yeah you can inherit members in a stock Enum -- that's one of the things that the `aenum` library adds (its written by the same dude who did the stdlib enum support).

I will loop back with some reference code after my next meeting

Reply:
What are the metric types that you want to provide?

We could change the `servo.types.Unit` to having an abstract base class that you could directly inherit from within the Wavefront connector instead of screwing with a new library.

I feel like we have a sorta chicken and egg problem where you don't quite know what I am talking about and I can't quite get your use-case fully in my head.

Reply:
So the unit types are equivalent, it's just the input strings are different. In sample configs Robert supplied me we havve things like the "request/m" vs. "rpm" and "percent" instead of "%". It's just the string format.